meeting_minutes/240307_SARIF_TC_85.md

```diff
@@ -0,0 +1,170 @@
+| Tim        | Hudson    | Cryptsoft Pty Ltd.      | Member                  | No      |
+| Vadim      | Okun      | NIST                    | Observer                | No      |
+
+Eight of the nine voting members present - quorum reached.
+
+Regrets from:
+
+* Michael Fanning
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* [Agenda for March 7, 2024](https://www.oasis-open.org/committees/download.php/71732/)
+
+The agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2024-02-03 Meeting #84](https://www.oasis-open.org/committees/download.php/71665/)
+
+The minutes were approved.
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on Charles to start the list of links to SARIF-related activities on the TC repo wiki
+  * ONGOING
+* ACTION on David Malcolm to create a first link list for SARIF producers and consumers
+  (from the wikipedia article draft on SARIF); David Malcolm spent some time on:
+  <https://github.com/oasis-tcs/sarif-spec/wiki/Known-Producers-and-Consumers>
+  See: <https://github.com/oasis-tcs/sarif-spec/wiki/Known-Producers-and-Consumers/_history> for the details of the edits.
+  * DONE
+* ACTION on Michael to organize label renovation
+  * ONGOING
+* ACTION on Michael to triage old issues
+  * ONGOING
+* ACTION on David Keaton to add subsection headings of Purpose and Objectives resp. to the future.md as a service to the reader
+  * DONE
+* ACTION on Nathan to add content to the future.md file https://github.com/oasis-tcs/sarif-spec/pull/626
+  * ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective voting members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+* Adar Weidman will regain voting rights after the next meeting in case he joins that meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+- Scheduled Teleconference (Thursday at 08:00 PT / **16:00** UTC for 1.5 hours)
+    ```
+    March 7, 2024
+    April 4, 2024 (note daylight savings time in effect, so Thursday at 08:00 PT / **15:00** UTC for 1.5 hours)
+    ```
+- Proposed Teleconference (Thursday at 08:00 PT / **15:00** UTC for 1.5 hours)
+    ```
+    May 2, 2024
+    ```
+
+# 3. Discussion
+
+## 3.1 Navigating the new OASIS web site for SARIF TC business
+
+* David Keaton walks shortly through the new website (workspace)
+
+## 3.2 Outreach
+
+### 3.2.1 Official SARIF logo?
+
+* David Keaton kindly asks if anyone knows about the logo on sarif.info if that can be shared with OASIS?
```

Comment:
I was asking if anyone had ideas for creating a SARIF logo.  The one on sarif.info is a placeholder.